client/devdocs/design/index.jsx

```diff
@@ -83,44 +97,44 @@ export default React.createClass( {
 					</SearchCard>
 				}
 				<Collection component={ this.props.component } filter={ this.state.filter }>
-					<Accordions />
```

Comment:
> However this creates a copy of each element in memory

Is this for DevDocs? I'm pretty sure performance won't be an issue, and if it is, we can address it, though it likely still won't be ultimately due to using `React.cloneElement`. That's not mangling the DOM or anything, just cloning a relatively small JavaScript object.